SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -339,6 +411,109 @@ TSharedRef<SWidget> FSpatialGDKEditorToolbarModule::CreateLaunchDeploymentMenuCo
 	return MenuBuilder.MakeWidget();
 }
 
+void OnLocalDeploymentIPChanged(const FText& InText, ETextCommit::Type InCommitType)
+{
+	if (InCommitType != ETextCommit::OnEnter && InCommitType != ETextCommit::OnUserMovedFocus)
```

Comment:
Does this ever work? 

Should it be: InCommitType != ETextCommit::OnEnter || InCommitType != ETextCommit::OnUserMovedFocus

Reply:
It would always return true if it was `||`.

Reply:
Ahh I am dumb I get it now...